packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/delete/delete.routing.js

```diff
@@ -0,0 +1,11 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.dedicatedClouds.datacenter.backup.delete', {
+    url: '/delete',
+    views: {
+      modal: {
+        component: 'dedicatedCloudBackupDelete',
+      },
+    },
+    layout: 'modal',
+  });
```

Comment:
No injection made ? Not even a `goBack` ?

Reply:
it's defined in the parent route, check the delete component.